pkg/manager/manager.go

```diff
@@ -165,11 +164,6 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 			return fmt.Errorf("unable to register webhooks to the manager error %v", err)
 		}
 
-		if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
```

Comment:
Is this change related?

Reply:
Yep, the HTTP has no use for us, it just say that the pod is up not that the webhook is ready, I have implement at "/readyz" at library so pod status will be ready when webhook is up and good it also checks that certs are good.